src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -21,7 +21,7 @@ interface IResourceLinkProps {
 
 const ResourceLink = (props: IResourceLinkProps) => {
   const { version } = props;
-  const { collections } = useAppSelector(state => state);
+  const { collections } = useAppSelector(state => state.collections);
```

Comment:
Hmm, shouldn't be de-structured?

Reply:
I don't think so, I only need the collections property in this case